packages/hix/lib/Hix/Data/NewProjectConfig.hs

```diff
@@ -29,7 +29,8 @@ data NewProjectConfigCommon =
   NewProjectConfigCommon {
```

Comment:
btw, a snappier name for this might be `CreateProjectConfig`, using the same scheme as in `local.nix`!